src/charts/BarChart.ts

```diff
@@ -0,0 +1,75 @@
+import { Chart } from 'chart.js'
+import { createCanvas } from 'canvas'
+import { ReplicationDistribution } from '../checker/Types'
+
+const RED = 'rgba(255, 99, 132)'
+const GREEN = 'rgba(75, 192, 192)'
+
+export default class BarChart {
+  private config: any
+  public readonly data: any
+  public readonly options: any
+
+  constructor(data: ReplicationDistribution[], opts: any = {}) {
```

Comment:
I am thinking of making this class generic so it can be reused for other types of data in the future.

Reply:
I also think we can just use static function since the class object is kind of stateless